google/api/resources.proto

```diff
@@ -44,8 +44,8 @@ message Resource {
 message ResourceSet {
   // The colloquial name of the resource.
   // If omitted, this is the name of the message.
-  string base_name = 1;
+  string name = 1;
 
   // The distinct resources that make up the set.
-  repeated Resource resources = 2;
+  repeated string resource_types = 2;
```

Comment:
"resource_types" should be "resource_references"

Reply:
Actually, I wanted to ask about that. (Sorry, I put a note in the doc, but you saw this first.)

I would like to keep the suffix as `type[s]` in order to match `google.api.operation`, which has `resource_type` and `metadata_type`, and accepts the same values. I believe that the "type" suffix is pretty important there and "reference" would be confusing, and that therefore this should match.

This is an opinion. Continue to challenge me if you disagree. :-)

Reply:
For visibility, Chris has replied: https://docs.google.com/a/google.com/document/d/1lavb-Tw5fU3Kp5AeVylri7jpLIpF7ILhb3PhyvnJszU/edit?disco=AAAACSEFKsc